src/components/arm/Arm.ts

```diff
@@ -1,43 +1,45 @@
-import type { Pose, WorldState } from '../../gen/common/v1/common_pb.esm';
+import type { WorldState } from '../../gen/common/v1/common_pb.esm';
```

Comment:
ideally we should define our own type for world state like we did for pose, assuming it's not too complicated of a definition

Reply:
discussed IRL, this struct is a bit more complicated so we can punt on this for now. there are a few other methods that expose protobuf structs in their signature so we can address when we change those